AppCenterData/AppCenterData/Internal/MSDocumentWrapperInternal.h

```diff
@@ -27,18 +26,19 @@
                                      eTag:(NSString *)eTag
                           lastUpdatedDate:(NSDate *)lastUpdatedDate
                          pendingOperation:(NSString *)pendingOperation
-                                    error:(MSDataError *)error
                           fromDeviceCache:(BOOL)fromDeviceCache;
 
 /**
  * Initialize a `MSDocumentWrapper` instance.
  *
  * @param error Document error.
  * @param documentId Document Id.
+ * @param partition Partition key.
+ * @param eTag Document eTag.
  *
  * @return A new `MSDocumentWrapper` instance.
  */
-- (instancetype)initWithError:(MSDataError *)error documentId:(NSString *)documentId;
+- (instancetype)initWithError:(MSDataError *)error documentId:(NSString *)documentId partition:(NSString *)partition eTag:(NSString *)eTag;
```

Comment:
Is the order of the parameters reviewed/finalized with discussion? Seems it should be like
```suggestion
- (instancetype)initWithError:(MSDataError *)error partition:(NSString *)partition documentId:(NSString *)documentId eTag:(NSString *)eTag;
```

Reply:
@jaelim-ms good idea, that'll be more consistent. Looks like I'll be adding more parameters anyway, and I'll make those consistent with the order of `initWithDeserializedValue`

Reply:
FYI The order might change in the future but +1 for consistency. It might change because `initWithDeserializedValue` was introduce at the time when the `read`, `create`, `replace` APIs from MSData.h first parameter was a partition.